webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -1051,20 +1051,21 @@ <h3>minidump-stackwalk output</h3>
               <table class="data-table">
                 <thead>
                   <tr>
-                    <th scope="col">Extension</th>
                     <th scope="col">Extension Id</th>
+                    <th scope="col">Name</th>
                     <th scope="col">Version</th>
-                    <th scope="col">Current?</th>
```

Comment:
The processor rule for processing the "Add-ons" annotation value only gets an id and a version--nothing about "current". I don't really know what this field intended.